ironfish/src/wallet/account.ts

```diff
@@ -583,6 +578,10 @@ export class Account {
 
     return assetBalanceDeltas
   }
+  spendingAccount(): SpendingAccount {
+    Assert.isTruthy(this.spendingKey, 'The provided account is not a spending account')
```

Comment:
I don't think it makes a practical difference, but I was a bit surprised to see `isTruthy` instead of `isNotUndefined`

Reply:
Seems reasonable to me and is more idiomatic